### ISSUE TITLE
Update ICRP growth curve

### ISSUE THREAD

**Issue by Erik-Sjogren** [Open-Systems-Pharmacology/Forum#87]
I have examined the Body Weight output for typical individuals (male) using the ICRP population in PK-Sim and following relationship between Body Weight and Age was attained.
 
![image](https://user-images.githubusercontent.com/29089721/31817294-a664af0e-b593-11e7-8e65-b9cdc22ccaef.png)

Going back to the ICRP reference one clearly see the origin of this curve as it overlaps with the reported values in Table 4.3 and Top Table at Page 64
  
![image](https://user-images.githubusercontent.com/29089721/31817369-f4c88b48-b593-11e7-8b7d-d6c7842e3901.png)
![image](https://user-images.githubusercontent.com/29089721/31817372-f7dbc85e-b593-11e7-9a5f-b992e8d92b37.png)

![image](https://user-images.githubusercontent.com/29089721/31817282-9ace0ac8-b593-11e7-81b9-ca1fa55c9ca1.png)

However, the ICRP reference contains additional information that can be included in the PK-Sim ICRP population to make the data complete (continuous) and hence more accurate.

Figure 4.1 at Page 59 displays the “Pattern of human growth, as described in this document”. Note that the shape of this growth curve is similar to several other reported populations (WHO, CDC, NHANES etc).
 
![image](https://user-images.githubusercontent.com/29089721/31817299-ad259d76-b593-11e7-9bf5-42ce4367af91.png)

 
Converting the Figure 4.1 data to body weight (assuming that a 25-year-old weighs approximately the same as a the tabulated “adult”  -> 100% mature=73 kg) and comparing that to PK-Sim and ICRP Table data gives: 
![image](https://user-images.githubusercontent.com/29089721/31817301-b15118b2-b593-11e7-9601-167d293870b1.png)

 
The data corresponds very well at the ages specified in the ICRP tables. This indicates that the assumption of 25 year=73 kg holds. However, there are deviations to PK-Sim, especially for the extrapolation between 15 and 30 years. 

Additionally, at Page 63 it is stated that:
_(78) Reference values for body mass are also based on the European data on body growth, together with consideration of the long-term increase in body mass after the apparent end of the growth period. Specifically, the reference value for body mass in the adult male is 10% greater than the central value determined for European males at age 18 years, and the reference value for adult females is 10% greater than that determined for European females at age 16 years._
This indicates that the male weight at age 18 yr is 73/1.1= 66.4 kg. Again, 73 kg=adult weight according to Table at Page 64
This calculated 18 yr weight clearly corresponds to the data in Figure 4.1.
 
![image](https://user-images.githubusercontent.com/29089721/31817332-cb70e556-b593-11e7-98c2-c8853683f154.png)

Furthermore, the deviations are carried on to virtual populations based on an ICRP individual in PK-Sim.
Below is a virtual population (n=5000) in the age range 0-35 years (grey dots) plotted together with the previous data.
![image](https://user-images.githubusercontent.com/29089721/31817340-d0f44f68-b593-11e7-8ba3-74579238ffa4.png)
  

So, is this of any importance? In my opinion the answer is, yes. Compared to the reported growth curve in ICRP (fig 4.1) the current PK-Sim ICRP model underestimate the body weight between 15-30 yrs. This may be of importance in certain analyses.

In conclusion, I think it is advisable to update the PK-Sim ICRP population growth according to the reported data.

Best,
E Sjögren

**Comment by msevestre** [comment]
@Erik-Sjogren: This is very interesting. Thanks for this very detailed post and explanations. 

@all: This is a great example of what open source is about. Contributions do not have to be code related. Model enhancements, critical analyses of existing features such as this one etc.. are fantastic ways to help improve the OSP Suite. 

Cheers,

**Comment by JanSchlender** [comment]
Hi Erik,

A really detailed and well-studied observation you provided here. Let me add a couple of points to this which may facilitate the discussion or explain at least the setting. 
1. One could clearly argue, if the development between 15 and 30years of age progresses linearly and if the ICRP population in the OSP fails to describe e.g. the 18year old adequately. However, the ICRP report for Europeans is the foundation for the respective population and there, the table on page 64 clearly states the body weight for a given age and 18years is not part of the table. The concept of the ICRP report is a holistic description of the human physiology at a couple of given ages so adding additional information only on body weight or any other parameter for a different age would cause trouble in scaling all other parameters. So when picking up your example, the “Create Individual” algorithm needs to add 7kg to an 18year old for which the ICRP report does not provide any information how to distribute it correctly. (You can study the algorithm in Willmann et al 2007 (https://www.ncbi.nlm.nih.gov/pubmed/17431751)). Since all other organs will be scaled linearly based on mainly body height between the two age bins, the remaining mass will be added to the fat mass. So by adding 7kg body weight, you would add 7kg fat mass and increase it by 2/3 (resulting in a higher fat mass than a 30year old).
2. The ICRP population represents a European population and should stand on its own. Data retrieved from the WHO would not be an adequate addition. This would encounter also ether ethnicities, for which dedicated populations should be used. You also mention the NHANES database for which the OSP provides a separate population. This or the Japanese population clearly have a different age trend.
So you see that by changing just one parameter, other parameters will be affected. In your last sentence you purpose to edit the ICRP population. I would disagree here and add for consideration that a population in a whole-body PBPK context is more than just anthropometric measures. Thus, I would leave the ICRP population as it is and purpose to build a new population with informed organ and tissue measures for the desired ethnicity. Guidance on this is provided in the literature (https://www.ncbi.nlm.nih.gov/pubmed/28401479, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5107207/). 
Best,
Jan

**Comment by Erik-Sjogren** [comment]
Thank you for the response Jan,

First to your second bullet. You misinterpret the aim of my post. I am not interested in creating a population of other ethnicity than ICRP or creating a mixed population. The only reference I made to other populations were the similarity in growth profiles between them and the data available for the ICRP. As given by the caption, the growth plot Fig 4.1 is “Based on central estimates for Western males as described in this document” hence the ICRP population. Fig 4.2 display the growth of US population (NHANES), but I have not used that data in any of my considerations.
![image](https://user-images.githubusercontent.com/29089721/32540152-0f243f46-c46c-11e7-92bf-3e4c0335e6f9.png)
OK, so let’s move on.

### Data in general
I do not share the viewpoint of being limited to the tabulated data when there is other data available (figure and text). In my opinion one should consider and evaluate all available data, based on the evaluation a rational and well-founded decision is then made to use the data or not. In this case I believe that, by the exclusion of this data the PK-Sim ICRP population will represent something else than the reported ICRP population. However, I will not force my opinion on anyone as there always will be different point of views. The statements and viewpoints that follows in this post are hence my personal.

### PK-Sim scaling
Thank you for the information regarding PK-Sim scaling of organ weights according to height for some organs, BWT for other and that the “rest BWT” is allocated to fat. I understand the rational of this approach to a certain level, even though I have some thoughts regarding this. There is for instance information of “mass of body fat” which enables another approach for body composition and mass distribution. However, reading the report by Willmann et al 2007 I see the intricate algorithms for population generation (i.e., ID variability). These should preferable be used in any extensions to the default populations. 

### Way forward
So, I have work out the following procedure to generate a new population (in this case it is an update to the ICRP) while preserving the population characteristics (i.e., variability) generated by PK-Sim. 

Any comments / thoughts to the general procedure described is much appreciated. 

a)	From the reported ICRP data the body weight (BWT)- and Height (HT)-to- age relationships was updated. I have already discussed the BWT data but I also found that the HT data needed a slight update. In PK-Sm the adult HT (176 cm) is attained at 30 years while the ICRP report that the adult Height is attained at age 18, virtually no growth occurs after this age which is logical.
 

b)	Updated BWT- and HT-age relationships for calculation of “new typical” (NT) values for these parameters were generated.
Similarly was relationships constructed based on the output from PK-Sim for the calculation of “old typical” (OT) values.

Below is the PK-sim and new (=Scaled) BWT-age relationships exemplified
 
![image](https://user-images.githubusercontent.com/29089721/32540189-340ac320-c46c-11e7-83ea-2904341723dd.png)


c)	A male ICRP population (n=10000) between 0-35 years was generated in PK-Sim and exported (CSV-file).
 
d)	 NT and OT values for BWT and HT were calculated for each ID in the population based on the IDs age and the above stated relationships.

e)	NTs and OTs were then used to scale each ID according to following equations. Where _old_ and _new_ indicate the old and new parameter value, respectively, for any given individual.

BWT_new= BWT_old × BWT_NT /BWT_OT
HT_new = HT_old × HT_NT /HT_OT
OrganVolume_new = OrganVolume_old × BWT_NT /BWT_OT
Organ.Specific.bloodflow_new = Organ.Specific.bloodflow_old × BWT_OT /BWT_NT × (HT_NT /HT_OT)0.75

f)	Parameters were updated with new values and the population can be imported in PK-Sim.

g)	By this approach the new BWT and HT are applied while maintaining the relative body composition and individual variability generated by PK-Sim. 
The blood flows are scaled according to the new HT but also compensated by any change in BWT to ensure that this transformation doesn’t affect total CO. 

Below are some plots showing the result of this exercise.


.
.

BWT output: PK-Sim ICRP male population 0-35 yr (n=10000)
 
![image](https://user-images.githubusercontent.com/29089721/32540194-3b488f5a-c46c-11e7-8d22-4d7011d8a945.png)


BWT output: Scaled PK-Sim ICRP male population 0-35 yr (n=10000)
 
![image](https://user-images.githubusercontent.com/29089721/32540204-3ed0273c-c46c-11e7-905b-6aaa5fd464fc.png)




Relative Fat volume (FatVolume/BWT): ICRP male population 0-35 yr (n=10000)
 
![image](https://user-images.githubusercontent.com/29089721/32540207-41870496-c46c-11e7-8af4-a601aa6bb415.png)

-

**Comment by Aedginto** [comment]
I am absolutely in favour of this. This is not adding additional information that ICRP doesn't provide and can still be called ICRP (for the sake of history it could be called ICPR_updated). Essentially, from what I can see, it adds two additional points but better describes Fig 4.1 in the ICRP. As long as all other balances are maintained (CO, FFM etc...), I vote yes:-)
Andrea

**Comment by tobiasK2001** [comment]
Dear Eric,

thnak you for this interesting points. I am also very much in favour of using all information available. The steps you take seem to be reasonable. So the only question I would have is: can you show that there is a significant difference in PK parameters when you simulate a drug with your new vs the current PK-sim Population. Do you see a significant change in AUC, CL, Vss etc. for your coumpound or some PK-sim template drug? Does your new population describe measured PK significantly better than the current one? If yes, I would aso vote yes.

Best, Tobias

**Comment by Aedginto** [comment]
I think that the decision to change the algorithm is irrespective of whether drug PK is differently simulated/predicted. The aim is to create the most relevant virtual people; only with this do we have increased confidence in the outcomes.

**Comment by Erik-Sjogren** [comment]
Thank you Andrea and Tobias for your positive responses.

Michael and Jan: Could you give some feedback on if the suggested approach is compatible with PKSim (both technically and for parameters). 

One concrete example with direct implications is pediatric scaling and comparing outcome using different methodologies (e.g. PBPK and PopPK). Then you really want to adopt the "same" population in both approaches, so to facilitate discrimination of method output. PopPK analyses are typically done with continuous age-bwt relationships which then are quite different than the age-bwt for the ICRP population in PK-Sim. 

Best,
Erik

**Comment by JanSchlender** [comment]
Hi Erik, great thoughts and elucidations you provided in both posts. Still, I guess, a couple of things need to be considered such as the implication of these changes on protein expressions, elimination pathways or how this matches with the development shown in the mass charts of the ICRP report for several organs. However, would you be willing to work on an implementation concept of this issue for the OSP and by this support the other platform users?
Best, Jan

**Comment by msevestre** [comment]
@Erik-Sjogren Once version 7.2 is out (very soon), I could provide with you with a special PKSim-DB version containing a clone of  the ICRP population (e.g.  ICRP Extended) and also a list of all population specific parameters that we have at the moment. After that, you could try to incorporate your findings and we could go from there?

**Comment by Erik-Sjogren** [comment]
Jan and Michael,
Sorry for the late response. 
Yes, I would gladly assist on this topic.
Best, Erik

**Comment by msevestre** [comment]
@Erik-Sjogren Fantastic. I propose to create a new repository where we shall continue our discussion. 
@JanSchlender @Yuri05 agreed?

**Comment by msevestre** [comment]
@Erik-Sjogren @JanSchlender 
I have created the following repository https://github.com/Open-Systems-Pharmacology/ICRP-Extended

Let's continue our discussion there
